### PR TITLE
Hw3 page objects

### PR DESCRIPTION
**Pull request by andrei-mikhailovskii**

### CODE REVIEW

File: src/test/java/com/epam/tc/hw3/page/IndexPageMainContainer.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    @FindBy(className = "icon-multi")
+    private WebElement iconMulti;
+
+    @FindBy(className = "icon-base")
+    private WebElement iconBase;
+
+    @FindBy(className = "benefit-txt")
+    private List<WebElement> listOfTexts;
+
+    @FindBy(xpath = "//div[normalize-space(span) = 'To include good practicesand ideas from successfulEPAM project']")
```
Comment (knadonenkoepam) [inline src/test/java/com/epam/tc/hw3/page/IndexPageMainContainer.java]: the same thing which you did in EX2, you don't need these elements, just one and verify that elements in list has the text

File: src/test/java/com/epam/tc/hw3/page/IndexPage.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        headerNavBar = new HeaderNavBar();
+        PageFactory.initElements(driver, headerNavBar);
+
+        leftNavBar = new LeftNavBar();
+        PageFactory.initElements(driver, leftNavBar);
+    }
+
+    public IndexPage loginUser(WebDriver driver, String userName, String userPassword) {
+        if (notLoggedInUserNames.size() != 0) {
+            new WebDriverWait(driver, WAIT_TIMEOUT_SEC)
```
Comment (knadonenkoepam) [inline src/test/java/com/epam/tc/hw3/page/IndexPage.java]: It is better to create a dedicated class which would be responsible for WebDriverWait

File: src/test/java/com/epam/tc/hw3/ex2/ExerciseTwoTest.java [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+import com.epam.tc.hw3.page.DifferentElementsPageMainContainer;
+import com.epam.tc.hw3.page.IndexPage;
+import com.epam.tc.hw3.page.RightSideBar;
+import org.assertj.core.api.SoftAssertions;
+import org.testng.annotations.Test;
+
+public class ExerciseTwoTest extends AbstractChromeTest {
+
+    private String correctPageTitle = "Home Page";
+    private String correctUserName = "ROMAN IOVLEV";
```
Comment (knadonenkoepam) [inline src/test/java/com/epam/tc/hw3/ex2/ExerciseTwoTest.java]: Username also could be moved and taken from resources

File: src/test/java/com/epam/tc/hw3/page/IndexPageMainContainer.java (hunk 1 above)
Comment (andrei-mikhailovskii) [reply src/test/java/com/epam/tc/hw3/page/IndexPageMainContainer.java]: removed excessive FindBy annotations with texts, and now a list of WebElements is used in "for" loop with getText method

File: src/test/java/com/epam/tc/hw3/page/IndexPage.java (hunk 2 above)
Comment (andrei-mikhailovskii) [reply src/test/java/com/epam/tc/hw3/page/IndexPage.java]: added WebDriverWaitClass with respective methods

File: src/test/java/com/epam/tc/hw3/ex2/ExerciseTwoTest.java (hunk 3 above)
Comment (andrei-mikhailovskii) [reply src/test/java/com/epam/tc/hw3/ex2/ExerciseTwoTest.java]: Moved Username to the config.properties file

**Review by knadonenkoepam — APPROVED** [review]

**Pull request merged**